examples/meteogram_metpy.py

```diff
@@ -0,0 +1,228 @@
+
+import numpy as np
+import matplotlib.pyplot as plt
+import pandas as pd
+import datetime as dt
+import matplotlib as mpl
+
+from metpy.units import units
+from metpy.calc import dewpoint_rh
+
```

Comment:
Need to add `from metpy.cbook import get_test_data`--sorry I forgot to mention that before.